output/json.go

```diff
@@ -0,0 +1,75 @@
+package output
+
+import (
+	"encoding/json"
+	"github.com/yarpc/crossdock/execute"
+	"github.com/yarpc/crossdock/plan"
+	"io/ioutil"
+)
+
+var StatusNames = map[execute.Status]string{
+	execute.Success: "success",
+	execute.Failed:  "failed",
+	execute.Skipped: "skipped",
+}
+
+type JSONTestReport struct {
```

Comment:
Generally, you want to avoid stuttering in exported types. Packages that import the `json` package will use this as `json.JSONTestReport`. It is completely acceptable to call it `TestReport` instead so that usage is `json.TestReport`. Same for `JSONBehaviorReport` and `JSONReport`.